client/app/home/home.html

```diff
@@ -31,18 +31,18 @@ <h1 class="section__title">
                     <footer>
                         <cite>Janet Chapman, Tanzania Development Trust</cite>
                     </footer>
-                    <p><a ng-href="/contribute?difficulty=ALL&organisation=Tanzania%20Development%20Trust">Find Tanzania Development Trust Projects</a></p>
+                    <p><a ng-href="/contribute?difficulty=ALL&text=Tanzania%20development%20trust">Find Tanzania Development Trust Projects</a></p>
                 </blockquote>
             </li>
             <li id="story-3" class="field-stories__card">
-                <h5>Rio Amazonas Flood Plain</h5>
+                <h5>Ongoing Disaster Mapping</h5>
                 <blockquote>
-                    <p>"Given the high risk of vector-borne diseases and damage to houses, the Peruvian Red Cross with the support of the German Red Cross are developing a mechanism, based on forecast information, to support families in a timely manner to conduct mitigation and preparedness actions to minimize these risks due to their vulnerability and exposure to floods."
+                    <p>"With our disaster mapping program, what we are trying to provide is a common operating picture for the first responders, aid organizations, governments and the affected people to make informed decisions. When all parties are working with the same information, the better situated the community is for quick and effective recovery."
```

Comment:
I thought I'd offer an idea for a very slight rewording of that (but just an idea. no need to hold up merging):

_"With our disaster mapping program, we aim to provide a common operating picture for the first responders, aid organizations, governments and the affected people to make informed decisions. When all parties are working with the same information, the community is better situated for quick and effective recovery."_

Reply:
Looks good to me.